### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by dima-cherednyk**
- [DEMO LINK](https://dima-cherednyk.github.io/js_get_data_DOM/)

### CODE REVIEW

File: src/scripts/main.js [hunk 1]
```diff
@@ -1,3 +1,23 @@
 'use strict';
 
-// write your code here
+const population = document.querySelectorAll('.population');
+
+const texts = [...population].map(amount => amount.innerHTML);
+
+const numbers = texts.map(x => +x.replace(/,/g, ''));
```
Comment (aliceblunt8) [inline src/scripts/main.js:7]: combine these 2 map methods into one

File: src/scripts/main.js [hunk 2]
```diff
@@ -1,3 +1,23 @@
...
+
+const averageToString = (totalAmount / numbers.length).toLocaleString('en-US');
+
+const total = document.querySelector('.total-population');
+
+const average = document.querySelector('.average-population');
+
+total.textContent = totalToString;
+
+average.textContent = averageToString;
```
Comment (aliceblunt8) [inline src/scripts/main.js:21]:
```suggestion
total.textContent = totalAmount.toLocaleString('en-US');
average.textContent = (totalAmount / numbers.length).toLocaleString('en-US');
```

File: src/scripts/main.js [hunk 3]
```diff
@@ -1,3 +1,23 @@
...
+
+// console.log(total, average);
```
Comment (aliceblunt8) [inline src/scripts/main.js:23]: remove console.log

File: src/scripts/main.js [hunk 4]
```diff
@@ -1,3 +1,23 @@
...
+
+const totalAmount = numbers.reduce((sum, x) => sum + x, 0);
+
+const totalToString = totalAmount.toLocaleString('en-US');
+
+const averageToString = (totalAmount / numbers.length).toLocaleString('en-US');
+
+const total = document.querySelector('.total-population');
+
+const average = document.querySelector('.average-population');
```
Comment (aliceblunt8) [inline src/scripts/main.js:17]: move all querySelector and querySelectorAll to the beginning of the file

**Review by aliceblunt8 — CHANGES_REQUESTED** [review]

**Review by aliceblunt8 — CHANGES_REQUESTED** [review]
fix comments from previous review or if you did this then push to Github, I don't see a new commit after review

**Comment by dima-cherednyk** [comment]
Problems with push.

**Comment by dima-cherednyk** [comment]
> 
fixed in dev01

**Pull request closed without merging**